Formula/srcml.rb

```diff
@@ -0,0 +1,223 @@
+class Srcml < Formula
+  desc "convert source code to XML"
+  homepage "http://www.srcml.org/"
+  url "http://131.123.42.38/lmcrs/beta/srcML-src.tar.gz"
```

Comment:
- Downloading from a numeric IP address is not something we encourage. We want long-lasting download URLs.
- The source code tarball should be tagged with version, otherwise we have no guarantee when upstream releases a new version that people install the version we advertise.
- Is it “beta” software or stable?